### PR TITLE
Fixed and updated 1und1.de ruleset.

### PR DESCRIPTION
**Pull request by Jochen-A-Fuerbacher**

### CODE REVIEW

File: src/chrome/content/rules/1und1.de.xml [hunk 1]
```diff
@@ -17,14 +10,20 @@ Fetch error: http://tt.1und1.de/ => https://tt.1und1.de/: (6, 'Could not resolve
...
 		- newsroom ²
 		- presse ²
+		- tt ⁵
 		- tv ³
 
 	¹ Dropped
 	² Refused
 	³ 404
+	⁴ Invalid certificate
+	⁵ Not found
```
Comment (J0WI) [inline src/chrome/content/rules/1und1.de.xml]: They have been removed from DNS, so we can remove them too.

File: src/chrome/content/rules/1und1.de.xml [hunk 2]
```diff
@@ -61,11 +60,11 @@ Fetch error: http://tt.1und1.de/ => https://tt.1und1.de/: (6, 'Could not resolve
 	¹ Secured by us
 
 -->
-<ruleset name="1und1.de (partial)" default_off='failed ruleset test'>
+<ruleset name="1und1.de (partial)">
 
 	<!--	Direct rewrites:
 				-->
-	<target host="apps.1und1.de" />
+	<target host="1und1.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/1und1.de.xml]: `account.1und1.de`

File: src/chrome/content/rules/1und1.de.xml [hunk 3]
```diff
@@ -17,14 +10,20 @@ Fetch error: http://tt.1und1.de/ => https://tt.1und1.de/: (6, 'Could not resolve
...
 	Nonfunctional subdomains:
 
+		- apps ⁴
+		- img ⁵
+		- login ⁴
 		- mirror ¹
 		- newsroom ²
 		- presse ²
+		- tt ⁵
 		- tv ³
```
Comment (J0WI) [inline src/chrome/content/rules/1und1.de.xml]: `erfolgscenter`
`service-center`

Comment (J0WI) [reply src/chrome/content/rules/1und1.de.xml]: Could you add them as comments since they have "Complications"?

Comment (Jochen-A-Fuerbacher) [reply src/chrome/content/rules/1und1.de.xml]: They are already added as comments at lines 29 and 31.

**Pull request merged**